### DISCUSSION TITLE
How to rotate AWS Cloudwatch log stream in every * Minute/Hour.

### DISCUSSION THREAD

**Discussion by pawanyoda** [fluent/fluent-bit#9403] [Q&A]
I was checking parameter rotate for cloudwatch log stream in every 5 minutes. I have seen in docs https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch but didn’t found any rotate kind of parameter. 
is there way to achieve this in fluentbit?

```
[Output]
    Name                cloudwatch_logs
    Match               *
    region              us-east-1
    log_group_name      /sky-tag-logs-dev
    log_stream_prefix   ${HOSTNAME}-
    auto_create_group   true
    log_retention_days  7
    extra_user_agent    container-insights
```

Thanks

**Comment by patrick-stephens** [comment]
Not sure I understand - what is rotating?

**Reply by pawanyoda** [reply]
I want rotate cloudwatch log stream in every 3 minute. Also fluentbit cloudwatch_logs plugin have an issue to re-create stream, If some how stream deleted its not creating automatically. Until restart fluentbit service.
